test ropetest.refactor.importutilstest.ImportUtilsTest.test_sorting_imports_and_standard_modules2 fails

Run

```
$ python -munittest -v \
ropetest.refactor.importutilstest.ImportUtilsTest.test_sorting_imports_and_standard_modules2
```

The test fails (with additional debugging logs):

```
matej@wycliff: rope (fix_ImportUtilsTest)$ python -munittest -v ropetest.refactor.importutilstest.ImportUtilsTest.test_sorting_imports_and_standard_modules2
test_sorting_imports_and_standard_modules2 (ropetest.refactor.importutilstest.ImportUtilsTest) ... DEBUG:test_sorting_imports_and_standard_modules2:pymod = dir [&lt;_ast.Import object at 0x240e5d0&gt;, &lt;_ast.Import object at 0x240e650&gt; ]
DEBUG:get_changed_source:first_non_blank = 3
DEBUG:get_changed_source:result = []
DEBUG:get_changed_source:sorted_imports = [&lt;rope.refactor.importutils.importinfo.ImportStatement object at 0x240e990&gt;, &lt;rope.refactor.importutils.importinfo.ImportStatement object at 0x240ea10&gt;]
DEBUG:get_changed_source:result = [u'import sys\n', '\n', u'import time\n']
DEBUG:get_changed_source:first_after_imports = 3
DEBUG:get_changed_source:result = [u'import sys\n', '\n', u'import time\n']
expected:
import sys
import time

observed:
import sys

import time

FAIL

======================================================================
FAIL: test_sorting_imports_and_standard_modules2 (ropetest.refactor.importutilstest.ImportUtilsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "ropetest/refactor/importutilstest.py", line 719, in test_sorting_imports_and_standard_modules2
    self.assertEquals(expected, observed)
AssertionError: 'import sys\nimport time\n' != u'import sys\n\nimport time\n'

----------------------------------------------------------------------
Ran 1 test in 0.039s

FAILED (failures=1)
matej@wycliff: rope (fix_ImportUtilsTest)$
```

It really looks to me like if the problem was with tests and the expected string.

--- a/rope/base/stdmods.py
+++ b/rope/base/stdmods.py
@@ -6,11 +6,14 @@ from rope.base import utils
 
 def _stdlib_path():
     import distutils.sysconfig
-    return distutils.sysconfig.get_python_lib(standard_lib=True)
+    return distutils.sysconfig.get_python_lib(standard_lib=True,
+                                              plat_specific=True)
+
 
 @utils.cached(1)
 def standard_modules():
     return python_modules() | dynload_modules()
+
 
 @utils.cached(1)
 def python_modules():
@@ -27,6 +30,7 @@ def python_modules():
                     result.add(name[:-3])
     return result
 
+
 @utils.cached(1)
 def dynload_modules():
     result = set(sys.builtin_module_names)
@@ -35,6 +39,8 @@ def dynload_modules():
         for name in os.listdir(dynload_path):
             path = os.path.join(dynload_path, name)
             if os.path.isfile(path):
-                if name.endswith('.so') or name.endswith('.dll'):
+                if name.endswith('.dll'):
                     result.add(os.path.splitext(name)[0])
+                if name.endswith('.so'):
+                    result.add(os.path.splitext(name)[0].replace('module', ''))
     return result

--- a/ropetest/builtinstest.py
+++ b/ropetest/builtinstest.py
@@ -476,6 +476,10 @@ class BuiltinModulesTest(unittest.TestCase):
         invalid = pymod['invalid'].get_object()
         self.assertTrue('sub' in invalid)
 
+    def test_time_in_std_mods(self):
+        import rope.base.stdmods
+        self.assertTrue('time' in rope.base.stdmods.standard_modules())
+
 
 def suite():
     result = unittest.TestSuite()

